net-im/discord/discord-0.0.84.ebuild

```diff
@@ -143,4 +143,10 @@ pkg_postinst() {
 	optfeature "sound support" \
 		media-sound/pulseaudio media-sound/apulse[sdk] media-video/pipewire
 	optfeature "emoji support" media-fonts/noto-emoji
+	if has_version -b kde-plasma/kwin[-screencast] && use wayland; then
+		einfo " "
+		einfo "On Wayland you need to enable kde-plasna/kwin[screencast] USE flag"
```

Comment:
If this is accurate, can't we just add a USE for `plasma` (which will be automatically set byPlasma profiles, and add a dep on `kde-plasma/kwin[screencast]` if USE has `+plasma +wayland`?

Also, do you have any documentation/links saying this is a required dependency?

Reply:
> If this is accurate, can't we just add a USE for `plasma` (which will be automatically set by Plasma profiles, and add a dep on `kde-plasma/kwin[screencast]` if USE has `+plasma +wayland`?
> 
> Also, do you have any documentation/links saying this is a required dependency?

Hey! Unfortunately it's a bug I caught by not being on plasma profile (I switched from desktop profile). I initially thought about the same idea to add if plasma && wayland as USE flags, but that would make discord depend on Kwin directly.

It's not necessarily a dependency, it's rather a 2nd grade dep due to the fact that kwin needs screencast support in order for discord to be able to share screen on Wayland. So it's not a direct dep.

Reply:
Ah, I see.

It seems we do something similar in the pipewire ebuild, so I guess it's not unprecedented.

@thesamesam Does the change seem same to with with your below review change applied? Not sure how we usually handle stuff like this.

Reply:
Yeah, it looks good to me. Thanks!

Reply:
@sarnex apologies, I might have closed the PR mistakenly.

Reply:
@sarnex https://github.com/gentoo/gentoo/pull/40446

My bad. I forgot Github closes PRs if you have one-on-one clone with master branch.